common/js/src/executable-module/executable-module-jstocxxtest.js

```diff
@@ -48,6 +48,38 @@ goog.exportSymbol('testExecutableModule', {
     }
   },
 
-  'testSmoke': function() {},
+  // Test that after the C++ code crashes via `GOOGLE_SMART_CARD_CHECK()`, the
+  // JS module object gets disposed of.
+  'testCrashViaCheck': async function() {
+    await testController.setUpCppHelper(
+        'LoggingTestHelper', /*helperArgument=*/ {});
+
+    try {
+      await testController.sendMessageToCppHelper(
+          'LoggingTestHelper', 'crash-via-check');
+      fail('Unexpectedly proceeded beyond crash');
```

Comment:
Oh just realized this is wrong. `fail()` tries to abort the test by throwing an exception, but my `try`-`catch` suppresses it immediately :flushed:

Reply:
Fixed. That was embarrassing.